source/includes/_introduction.md.erb

```diff
@@ -1,13 +1,29 @@
-# Introduction
+# Overview
+The SnapChance API provides access to Multilot's platform to create custom scratch games, manage lotteries, as well as information about the games and the players. It allows customer platforms to customize the user experience.
 
-The SnapChance API is developed to support entertainment games such as scratch card game which gives a player an opportunity to win money.
+The scratch game supports two types of winnings.
+* An instant win: The user is receives the winnings right at the end of the game.
+* A win at the end of a draw: Some prizes are rewarded at the end of a draw. The player won't receive these until the draw has ended.
 
-The API is secured with JWT authentication and the requests are authorized for the users with valid credentials. The user must authenticate with API and get a JWT token; which can be used to access all secured endpoints. The API has build-in integration with NETs for BankID verification and payment process.
+This documentation provides information about each endpoint in the API. The right-hand pane of this page shows example requests and responses for each endpoint in different languages.
 
-#### SnapChance API supports users with three roles:
-- <b>Regulator</b> - Create and manage the licenses.
-- <b>Operator/Admin</b> - Create and manage the lotteries.
-- <b>Player</b> - Add funds to the wallet; purchases and play tickets.
+
+# Getting started
+Follow these steps to start using the SnapChance API:
+
+* Sign contract with Multilot SnapChance
+* Get URLs and Credentials for test and production environment from Multilot SnapChance support team
+* Create an application.
```

Comment:
@justinhamadeunity This is similar to the VSP docs please let me know if you think this makes sense here